android/gradle.properties

```diff
@@ -19,4 +19,6 @@
 
 android.useDeprecatedNdk=true
 MYAPP_RELEASE_STORE_FILE=kiwi-rn-hotels.keystore
-MYAPP_RELEASE_KEY_ALIAS=kiwi-rn-hotels
\ No newline at end of file
+MYAPP_RELEASE_KEY_ALIAS=kiwi-rn-hotels
+android.enableAapt2=false
```

Comment:
This was added because of this issue
https://github.com/facebook/react-native/issues/16906

Reply:
The better fix is to use the Gradle file from the PR in the issue you linked, and apply that one for the time being. It was mentioned that not using AAPT2 could cause some older devices to crash!

Reply:
@mathieumg, thanks for pointing this out. I think in our case, it it still ok. We are currently just providing a bundle to be consumed by our native app. 

I build this apk to test performance with production build.